app/frontend/controllers/tab_controller.js

```diff
@@ -516,7 +487,7 @@ class TabController {
     return selectedTabsPanelId;
   }
 
-  addTab(metaTab=undefined, interactive=true, bibleTranslationId=undefined) {
+  addTab(metaTab = undefined, interactive = true, bibleTranslationId = undefined) {
```

Comment:
This is minor, but we should discuss this code style aspect eventually ...
I think having no space to the left and right of default parameters in functions is quite common.